src/Projekt/Projekt.fsproj

```diff
@@ -81,16 +57,54 @@
   <Target Name="BeforeBuild">
   </Target>
   -->
-  <ItemGroup>
-    <Templates Include="$(SolutionDir)\templates\**\*.*"/>
-  </ItemGroup>
   <Target Name="AfterBuild">
-    <Copy
-        SourceFiles="@(Templates)"
-        DestinationFolder="$(OutputPath)\templates\%(Templates.RecursiveDir)"
-        />
+    <Copy SourceFiles="@(Templates)" DestinationFolder="$(OutputPath)\templates\%(Templates.RecursiveDir)" />
```

Comment:
So this is one of those times where I begin to realize how much I don't know.  Does msbuild not evaluate the Imported elements with the same level of rigor as top-level nodes?